src/metatensor/models/experimental/soap_bpnn/train.py

```diff
@@ -48,20 +48,29 @@ def train(
     continue_from: Optional[str] = None,
     output_dir: str = ".",
 ):
+    # TODO: check that we're only training on energy targets.
+    # also in alchemical
+
     all_species = get_all_species(train_datasets + validation_datasets)
     outputs = {
         key: ModelOutput(
             quantity=value.quantity,
             unit=value.unit,
-            per_atom=False,
+            per_atom=True,
         )
         for key, value in dataset_info.targets.items()
     }
+    # the model is always capable of outputting the last layer features
+    outputs["mts-models::aux::last_layer_features"] = ModelOutput(
+        quantity="",
+        unit="",
+        per_atom=True,
+    )
     new_capabilities = ModelCapabilities(
         length_unit=dataset_info.length_unit,
         outputs=outputs,
         atomic_types=all_species,
-        supported_devices=["cpu", "cuda"],
+        supported_devices=["cuda", "cpu"],
```

Comment:
are you sure you wanna default to cuda first here? Since this is based on rascaline which does not support cuda fully

Reply:
Yes, I still found this model to be faster on CUDA. We can benchmark properly in the future if you want